initializr-service/src/main/resources/application.yml

```diff
@@ -971,6 +971,17 @@ initializr:
           versionRange: 1.3.0.RELEASE
           groupId: org.springframework.cloud
           artifactId: spring-cloud-stream-binder-kafka
+        - name: Stream Reactor
+          id: cloud-stream-reactive
+          description: Messaging microservices with Reactor. Typically used with a Spring Cloud Stream binder implementation.
```

Comment:
>  Typically used with a Spring Cloud Stream binder implementation.

What does that mean. Does this work standalone or does this require another dependency?

Reply:
it requires another dependency. You must choose a Spring Cloud Stream binder implementation like Kafka or RabbitMQ